### DISCUSSION TITLE
Refund Request

### DISCUSSION THREAD

**Discussion by linzibrynn45** [aclap-dev/video-downloadhelper#1042] [Help and Q&A]
### Your question

I’ve been working with a few users on here to figure out my problem but no one can figure out why it is happening. I’m kindly requesting a refund. Thank you. 

Link to the issue. 
https://github.com/aclap-dev/video-downloadhelper/discussions/973#discussion-6899504  

### Addon & CoApp version and details

It is in the link above. #973

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
